jablib/build.gradle.kts

```diff
@@ -496,3 +476,19 @@ jacocoTestReport {
     }
 }
 */
+
+javaModuleTesting.whitebox(testing.suites["test"]) {
+    requires.add("io.github.classgraph")
+    requires.add("org.junit.jupiter.api")
+    requires.add("org.junit.jupiter.params")
+    requires.add("org.jabref.testsupport")
+    requires.add("org.mockito")
+
+    // --add-opens
+    opensTo.add("javafx.base/com.sun.javafx.beans=net.bytebuddy")
+    opensTo.add("javafx.fxml/javafx.fxml=org.jabref")
```

Comment:
The `opensTo` here is oriented at what you can do in `module-info.java` files and not at the command line. The idea is to have this "block" look similar to how you would write a `module-info.java` if it were possible for whitebox tests.

The `opensTo` opens **all** packages of **this** test module (tests of `org.jabref.jablib`) to another module.

E.g. by default the plugin already does `opensTo.add("org.junit.platform.commons")`.

You can consistently open packages in the existing modules `javafx.base` of `javafx.fxml` by patching them. [See my comment here.](https://github.com/gradlex-org/extra-java-module-info/issues/178#issuecomment-2878744333)